users.js

```diff
@@ -272,6 +272,7 @@ class User {
 		this.registered = false;
 		this.userid = toId(this.name);
 		this.group = Config.groupsranking[0];
+		this.lastActive = new Map();
```

Comment:
Well, this system is kind of a per-room thing, currently.  I figure it would be best to do it this way for accuracy if another room has slowchat enabled as well.

With that being said, I'm not sure what would be the best way to do it then.  What are your thoughts?